node/node.go

```diff
@@ -869,11 +880,16 @@ func (node *Node) StartMining(ctx context.Context) error {
 	go node.handleNewMiningOutput(miningCtx, outCh)
 
 	// initialize a storage miner
-	storageMiner, err := initStorageMinerForNode(ctx, node)
+	storageMiner, ownerAddress, err := initStorageMinerForNode(ctx, node)
```

Comment:
BLOCKING: This needs to be the worker address. This probably just means renaming this variable here and elsewhere for now.